system-test/compute.js

```diff
@@ -21,13 +21,25 @@ const compute = require('../');
 
 const DiskType = compute.protos.google.cloud.compute.v1.AttachedDisk.Type;
 
+const delay = async test => {
+  const retries = test.currentRetry();
+  if (retries === 0) return; // no retry on the first failure.
+  // see: https://cloud.google.com/storage/docs/exponential-backoff:
```

Comment:
It's odd to have a link of storage not compute here. This link might be more appropriate here. https://github.com/grpc/proposal/blob/master/A6-client-retries.md#exponential-backoff